### PR TITLE
Calendar subscription

### PR DESCRIPTION
**Pull request by milt**
Adds crud events for calendar subs

### CODE REVIEW

File: src/transformer/events/core/calendar_subscription_updated.php [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+ * @return array
+ */
+
+function calendar_subscription_updated(array $config, \stdClass $event) {
+    $repo = $config['repo'];
+    $user = $repo->read_record_by_id('user', $event->userid);
+    $course = $event->courseid == 0 ? null : $repo->read_record_by_id('course', $event->courseid);
+    $lang = is_null($course) ? $config['source_lang'] : utils\get_course_lang($course);
+    $subscription = $repo->read_record_by_id('event_subscriptions', $event->objectid);
+    $object_id = $config['app_url'].'/calendar/subscription?id='.$subscription->id;
```
Comment (cliffcaseyyet) [inline src/transformer/events/core/calendar_subscription_updated.php]: i thought we were mostly centralizing id logic inside the object creators. Seems like potential for conflict if not.

**Review by cliffcaseyyet — APPROVED** [review]
approved but one minor suggestion that i dont care passionately about.

File: src/transformer/events/core/calendar_subscription_updated.php (hunk 1 above)
Comment (milt) [reply src/transformer/events/core/calendar_subscription_updated.php]: yeah I can refactor that out

Comment (milt) [reply src/transformer/events/core/calendar_subscription_updated.php]: https://github.com/yetanalytics/moodle-logstore_xapi/pull/67/commits/12c87c067eaaf65f9f31732c9622e96d71383128

**Pull request merged**